lib/CodeGen/CGExprScalar.cpp

```diff
@@ -1597,7 +1575,7 @@ Value *ScalarExprEmitter::VisitCastExpr(CastExpr *CE) {
   }
 
   case CK_IntToOCLSampler:
-    return CGF.CGM.createOpenCLIntToSamplerConversion(E, CGF);
+    return CGF.CGM.createOpenCLIntToSamplerConversion(E, CGF);
```

Comment:
So, I worked out what this change is. It's not whitespace exactly, it's the wrong kind of line ending (this changes the ending from windows style to unix style). Let's leave it in this fix so it doesn't appear every time we change this file.